neo/Consensus/RegenerationMessage.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.IO;
+using System.Linq;
+using Neo.Network.P2P.Payloads;
+using Neo.IO;
+
+namespace Neo.Consensus
+{
+    internal class Regeneration : ConsensusMessage
+    {
+        /// <summary>
+        /// Original PrepareRequest in which, at least, M nodes signed
+        /// </summary>
+        public ConsensusPayload PrepareRequestPayload; 
+        /// <summary>
+        /// Partial signatures of, at least, M nodes
+        /// </summary>
+        public byte[][] SignedPayloads;
+
+        public override int Size => base.Size + PrepareRequestPayload.Size + SignedPayloads.Length;
```

Comment:
This `Size` is wrong calculated, `SignedPayloads.Length` is not the right method to compute this length

Reply:
Right, this is the issue we discussed on chat, what is the correct calculation?

Reply:
done here: https://github.com/neo-project/neo/pull/426/commits/dab61a188f8baf8b8ac9a184e435072473820161

![image](https://user-images.githubusercontent.com/3167973/48836132-2eb5d500-ed82-11e8-860b-4640e6253f5a.png)

Reply:
Thanks for fixing.

Reply:
What tools do you use? It surely helps